### DISCUSSION TITLE
Is it safe to run multiple renovate instances on the same host?

### DISCUSSION THREAD

**Discussion by konrad-ohms** [renovatebot/renovate#14727] [Request Help] [answered]
Hi,
I am running the self-hosted renovate slim container image on a VM. I have Jenkins in place with multiple executors on the same worker node and schedule each repository update individually. I ensure that only a single renovate instance is working on the same repo at the same time via Jenkins locks, but as I use binarySource=docker and mount the docker socket into the renovate container, I am wondering if that is safe to do. I noticed that renovate tries to clean dangling containers, could that overlap between multiple executions?

As I run renovate on quite a few repos, I would like Jenkins to run the updates in parallel.

Would it be possible to disable the cleanup in renovate and instead to manage it separately, e.g. periodic cleanups via Jenkins? 

Thanks in advance
Konrad

**Comment by viceice** [comment, marked as answer]
Yes, you need to make sure renovate doesn't use same work directory (baseDir) 

You also need to configure this:

https://docs.renovatebot.com/self-hosted-configuration/#dockerchildprefix

**Reply by konrad-ohms** [reply]
Ah great, Thanks a lot for your fast response
